Update SP connections config (#4343)
Particular/docs.particular.net: servicepulse/host-config.md

Change the value after `--url=` and restart the service.

> ## Connection to ServiceControl and ServiceControl Monitoring

Reviewer: ```suggestion
## Connecting to ServiceControl and ServiceControl Monitoring
```

> ServicePulse will, by default, attempt to connect to ServiceControl and ServiceControl Monitoring using the URLs `http://localhost:33333/api/` and `http://localhost:33633/`, respectively.

Reviewer: ```suggestion
ServicePulse will by default, attempt to connect to ServiceControl and ServiceControl Monitoring using the URLs `http://localhost:33333/api/` and `http://localhost:33633/`, respectively.
```

### Configuring connections via the ServicePulse UI

The default connections can easily be overriden via the Connections configuration screen in ServicePulse:

![Connections configuration](images/connections-configuration.png 'width=500')

> To configure connections via the Connections configuration screen, the ServiceControl (Monitoring) URLs need to be entered, with the port number included. This configuration will persist only locally and is read in the following order:

Reviewer 2: ```suggestion
To configure connections via the Connections configuration screen, the ServiceControl and ServiceControl Monitoring full URLs need to be entered, including port number and scheme. Connection configuration is persisted only locally and is read in the following order:
```

> 1. **URL in the browser address bar** - allows easily bookmarking and sharing connection details

Reviewer 2: ```suggestion
1. **Query string in the browser address bar** - allows easily bookmarking and sharing of connection details.
```

> 1. **Query string in the browser address bar** - allows easily bookmarking and sharing of connection details.

Author: QueryString 👍 

Browser's address bar makes it precise.

> 1. **Query string in the browser address bar** - allows easily bookmarking and sharing of connection details.

Author: Actually, according to https://en.wikipedia.org/wiki/Query_string query string is 2 words, not one.

> 1. **Query string in browser address bar URL** - allows easily bookmarking and sharing of connection details.

Reviewer 3: @mauroservienti @HEskandari @WilliamBZA I was led here while reviewing https://github.com/Particular/docs.particular.net/pull/4394. I'm not sure I understand this. Does this mean that, when I switch to another ServiceControl instance, the address to ServiceControl is then embedded in the URL for ServicePulse?

Reviewer 2: Correct.

> 2. **Local storage** - remembers the last used connection if connection parameters are later not found in the URL

Reviewer 2: ```suggestion
2. **Local storage** - remembers the last used connection if connection parameters are later not found in query string.
```

Author: According to https://en.wikipedia.org/wiki/Query_string a query string goes inside a URL. I modified to make it "query string in URL"

> 2. **Local storage** - remembers the last used connection if connection parameters are later not found in the URL

Reviewer 4: Maybe this?

>If the connection parameters are not found in the URL, the last remembered connection will be used.

Author: I turned that suggestion into something slightly better:

> If the connection parameters are not found in the URL, the last successful connection will be used.
